addons/mass_mailing/models/mailing.py

```diff
@@ -218,9 +218,9 @@ def _get_default_mail_server_id(self):
     failed = fields.Integer(compute="_compute_statistics")
     received_ratio = fields.Integer(compute="_compute_statistics", string='Received Ratio')
     opened_ratio = fields.Integer(compute="_compute_statistics", string='Opened Ratio')
-    replied_ratio = fields.Integer(compute="_compute_statistics", string='Replied Ratio')
+    replied_ratio = fields.Float(compute="_compute_statistics", string='Replied Ratio')
     bounced_ratio = fields.Integer(compute="_compute_statistics", string='Bounced Ratio')
-    clicks_ratio = fields.Integer(compute="_compute_clicks_ratio", string="Number of Clicks")
+    clicks_ratio = fields.Float(compute="_compute_clicks_ratio", string="Number of Clicks")
```

Comment:
Why don't you update other ratio fields ? Any reason replied and clicks are float and not the other one ?

Reply:
Hello @tde-banana-odoo 

In spec, it was there for these two fields only

Thanks

Reply:
Mmmh ... ok then, as it is not stored it is just a matter of decimals.